src/soroban.js

```diff
@@ -0,0 +1,74 @@
+import BigNumber from 'bignumber.js';
+
+/**
+ * Soroban helper class
+ * formatting, parsing, and etc
+ * @class Soroban
+ */
+export class Soroban {
+  /**
+   * format token amount to its ideal specific number of decimals
+   *
+   * @param {BigNumber} amount - the token amount you want to display
+   * @param {number} decimals
```

Comment:
needs a description

Reply:
should it default to 7?

Reply:
IMHO we should not provide a default here. Even though we might be able to guess that folks will need 7 the most often I think having a default makes it easier to overlook the arg and make a mistake.

Reply:
yep, fair!